canonical-websites/www.ubuntu.com
# update /core to vbt

## Done

* update /core to vbt
* created a pattern for logo clouds

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/core)
- Compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/6.%20Core?preview=core-overview.png) and the [demo](http://www.ubuntu.com-1598-core-overview.demo.haus/core)

## Issue / Card

Fixes #1598
